### ISSUE TITLE
Spigotmc.org Direct Download Link to Latest Spigot Jar

### ISSUE THREAD

**Issue by AlexMcDermott** [GeyserMC/Geyser#3085]
### What feature do you want to see added?

On the Spigotmc page it would be handy to have the download link directly download the latest spigot jar so that its compatible with this great docker image https://hub.docker.com/r/itzg/minecraft-server. The image is able to automatically download plugins if a direct link is present as seen here: https://github.com/itzg/docker-minecraft-server#auto-downloading-spigotmcbukkitpapermc-plugins. Atm on your Spigotmc it just links to https://ci.opencollab.dev//job/GeyserMC/job/Geyser/job/master/ so the jar needs to be downloaded manually as the image can't fetch it automatically using the Spigotmc API. I'm only just starting to look at at this but seems like most people on Spigotmc would be after the Geyser-Spigot.jar anyway. The feature I'm proposing could be easily implemented by setting the download button on Spigotmc to this link: https://ci.opencollab.dev//job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar

### Are there any alternatives?

Atm I'm just downloading it with wget manually before starting the server but this isn't as robust and would be nice to have it all wrapped up in the container. Having it in the container would also just allow for the docker restart functionality to auto be used instead of a script on restart to download the plugin.

**Comment by Jens-Co** [comment]
Hello there,

due to the type of plugin Geyser is, this is not something we are keen in doing direct downloads to spigot page. 
Geyser updates a lot and uploading latest version each time to spigot is kind of a time waste. 

Nonetheless there is a plugin (https://github.com/kejonaMC/GeyserUpdater) which is an unaffiliated third party plugin that is able to update Geyser/branches without any issues. 

Jens

**Comment by Camotoy** [comment]
Yeah, we won't be implementing this also because we want to be able to link all the Geyser versions from there. There are other options as Jens as mentioned. Thanks anyway!

**Comment by AlexMcDermott** [comment]
That's fair, I wasn't aware that it was up to you to upload the jar to Spigotmc I thought you could give them a download link that they'd link to. Anyway all good, I found a workaround that still uses the same docker image. It uses the MOD env variable documented here (https://github.com/itzg/docker-minecraft-server#downloadable-modplugin-pack-for-forge-fabric-and-bukkit-like-servers). Just for reference if anyone else sees this.
